File: Examples/test-suite/perl5/li_cpointer_runme.pl
use strict;
use warnings;
use Test::More tests => 3;
BEGIN { use_ok('li_cpointer') }
require_ok('li_cpointer');

my $p = li_cpointer::new_intp();
li_cpointer::intp_assign($p, 3);
is(li_cpointer::intp_value($p), 3);
li_cpointer::delete_intp($p);

File: Examples/test-suite/javascript/li_cpointer_runme.js
var li_cpointer = require("li_cpointer");


p = li_cpointer.new_intp();

if (li_cpointer.intp_value(p) != 0) {
    throw new Error("not initialized");
}

li_cpointer.intp_assign(p, 3);

if (li_cpointer.intp_value(p) != 3) {
    throw new Error;
}

li_cpointer.delete_intp(p);
